src/ReactDataGrid.js

```diff
@@ -369,33 +366,37 @@ const ReactDataGrid = React.createClass({
     }
   },
 
+  isCellWithinBounds(cell) {
```

Comment:
FYI you can make this simpler with ES6.

Using object destructuring in the params.

```
isCellWithinBounds({ idx, rowIdx }) {
   return idx >= 0
      && rowIdx >= 0
      && idx < ColumnUtils.getSize(this.state.columnMetrics.columns)
      && rowIdx < this.props.rowsCount;
}
```